src/components/Post.js

```diff
@@ -189,11 +194,18 @@ export class CommentSection extends React.Component {
     dispatch: func
   }
 
-  static contextTypes = {post: object, dispatch: func, currentUser: object}
+  static contextTypes = {
+    post: object,
+    dispatch: func,
+    community: object,
+    currentUser: object
+  }
 
   render () {
     let { comments, truncate, expand } = this.props
-    const { dispatch, post, currentUser } = this.context
+    const { dispatch, post, currentUser, community } = this.context
+    let communitySlug = community ? community.slug : ''
```

Comment:
can you use lodash.get here and drop the case where you set the slug to an empty string? the util functions already deal with null values of slug, so it's a weird signal to the reader to set an empty string instead of null